### PR TITLE
Query and update animals table

### PR DESCRIPTION
**Pull request by shahira-sadat**
In this pull request I did 👇🏻: 

- Add `species` column to the `animals` table
- Add more data to the `animals` table.
- Handle transactions
- Create more queries to get data from the `animals` table and answer questions

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -14,4 +14,79 @@ SELECT * FROM animals WHERE neutered = true;
...
+
+/* Part 1 */
+
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
+ROLLBACK;
+
```
Comment (mikethreels) [inline queries.sql:27]: - [x] you should verify if the changes were made before and after rollback as well

File: queries.sql [hunk 2]
```diff
@@ -14,4 +14,79 @@ SELECT * FROM animals WHERE neutered = true;
...
+BEGIN;
+
+UPDATE animals
+SET species = 'digimon' WHERE name LIKE '%mon';
+
+UPDATE animals
+SET species = 'pokemon' WHERE species IS NULL;
+
+COMMIT;
+
```
Comment (mikethreels) [inline queries.sql:39]: - [x] you should verify if the changes were made before and after commit as well

File: queries.sql [hunk 3]
```diff
@@ -14,4 +14,79 @@ SELECT * FROM animals WHERE neutered = true;
...
+/* Part 3 */
+
+BEGIN;
+
+DELETE FROM animals;
+
+ROLLBACK;
+
```
Comment (mikethreels) [inline queries.sql:47]: - [x] you should verify if the changes were made before and after rollback as well

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Shahira, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
- [x] professional looking readme

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (shahira-sadat) [reply queries.sql:27]: Done ✅

File: queries.sql (hunk 2 above)
Comment (shahira-sadat) [reply queries.sql:39]: Done ✅

File: queries.sql (hunk 3 above)
Comment (shahira-sadat) [reply queries.sql:47]: Done ✅

**Review by KDlamini — APPROVED** [review]
### Hi @shahira-sadat 🙋‍♀️,  

Excellent work! 💯 
You have done a great job implementing all the required changes. 🎊 🥳 
Your feature is complete! There is nothing else to say other than... it's time to merge it. :shipit: 
Congratulations!!!

## HIGHLIGHTS 🌟 

✔️ All SQL for schema, data, and query files work as expected

# Status: Approved ✔️ 

simon

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**